### ISSUE TITLE
gym (0.9.3),,ImportError: No module named client, from gym.scoreboard.client

### ISSUE THREAD

**Issue by dialogchat** [ppaquette/gym-super-mario#17]
when running this basic import code I get error below

```
import gym
import gym_pull
gym_pull.pull('github.com/ppaquette/gym-super-mario')        # Only required once, envs will be loaded with import gym_pull afterwards
env = gym.make('ppaquette/SuperMarioBros-1-1-v0')
```

Traceback (most recent call last):
  File "super_mario_1.py", line 2, in <module>
    import gym_pull
  File "/usr/local/lib/python2.7/dist-packages/gym_pull/__init__.py", line 40, in <module>
    import gym_pull.scoreboard.api
  File "/usr/local/lib/python2.7/dist-packages/gym_pull/scoreboard/api.py", line 3, in <module>
    from gym.scoreboard.client import resource, util
ImportError: No module named client

**Comment by Zman94** [comment]
I have the same issue. Have you found a solution?

**Comment by YunqiuXu** [comment]
Same problem

[Solved] use gym 0.7.4, after installing gym-pull, you will find gym is upgraded to 0.9.3

**Comment by Zman94** [comment]
I was able to solve it by installing the package with ``python setup.py install` then importing it with `import ppaquette_gym_super_mario`

**Comment by UesugiErii** [comment]
I installed it according to the following steps:
    1.Uninstall your gym if the version not equal 0.7.4
    2.Install gym(0.7.4)  by command (sudo pip3 install gym==0.7.4)
    3.Download or git clone code
    4.Manual installation this by command (sudo python3 setup build)&(sudo python3 setup install)
</br>
This is more than a gym-mario library,I see another(link: https://github.com/Kautenja/gym-super-mario-bros).
The latter is easier to install,but this can be more customized(such as can accept 64 kinds of action,the latter only can accept 14,and this can select a level)

**Comment by ppaquette** [comment]
Fixed in 0.0.8
